File: HotelVillaApplication/app/src/main/java/be/cienfuegos/sandra/hotel_villa_application/api/model/Address.java
package be.cienfuegos.sandra.hotel_villa_application.api.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
public class Address {
    private Country country;
    private int countryId;
    private String lineOne;
    private String lineTwo;
    private String postCode;
    private String city;
}

File: API/API/Domain/Models/Address.cs
using System;
using Microsoft.EntityFrameworkCore.Storage;

namespace API.Domain.Models
{
    public class Address
    {
        public int AddressId { get; set; }

        public int CountryId { get; set; }

        public Country Country { get; set; }

        public string LineOne { get; set; }

        public string LineTwo { get; set; }

        public string PostCode { get; set; }

        public string City { get; set; }

        public Customer Customer { get; set; }
    }
}